0.X/Dockerfile

```diff
@@ -46,6 +46,10 @@ RUN mkdir -p /consul/data && \
     mkdir -p /consul/config && \
     chown -R consul:consul /consul
 
+# set up nsswitch.conf for Go's "netgo" implementation which is used by Consul,
+# otherwise DNS supercedes the container's hosts file, which we don't want.
+RUN [ ! -e /etc/nsswitch.conf ] && echo 'hosts: files dns' > /etc/nsswitch.conf
```

Comment:
Wont this eval to a non-zero exit code if for some reason a newer alpine were to include an nsswitch.conf

I think it would be better as `RUN test -e /etc/nsswitch.conf || echo 'hosts: files dns' > /etc/nsswitch.conf` that way the only way the command fails is if nsswitch.conf didn't exist and wasn't created.